### PR TITLE
Add cbt_ratings.py backed by cbt

### PR DESCRIPTION
**Pull request by sethtroisi**
This still needs a little work before it can completely replace ratings.py

```
export PROJECT=<project>
source cluster/common.sh

# Needed for workstations
export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=/usr/share/grpc/roots.pem

sqlite3 cbt_ratings.db < ratings/schema.sql
time python3 ratings/cbt_ratings.py  --cbt_project "$PROJECT" --cbt_instance "$CBT_INSTANCE" --sync_ratings=True
```

```
8217it [00:02, 4007.72it/s]
Existing models(cbt): 8217
New models insertted into db: 8217
eval_game: 1084094 rows [01:25, 12729.42 rows/s]

Inserted 73584 new games from 73584 rows
Wins(73584) updated
Models(1001) updated

168501     considered
94917      determine failed
DB has 73584 games

Best models
            v17/000820-defence: (3712.7669732848617, 37.47806885739926)
            v17/000961-artemis: (3702.949168673891, 34.874364073248806)
             v17/000960-diadem: (3695.781720209018, 37.05316482411292)
            v17/000934-neptune: (3692.3666041951424, 37.689314891704285)
            v17/000750-modeste: (3691.4547645739326, 36.84205596563322)
           v17/000862-vigilant: (3679.9932617308054, 36.806393837995635)
             v17/000848-europa: (3676.990532789002, 36.75761040659639)
             v17/000891-sultan: (3676.56959823727, 36.49635029095474)
             v17/000917-europa: (3663.473566405014, 36.25758522079187)
             v17/000797-portia: (3658.1404930018407, 36.96180115585781)
              v17/000915-grief: (3656.2379638117204, 36.04500084775167)
           v17/000767-wanderer: (3652.5521839855487, 36.825015865158484)
           v17/000903-hastings: (3648.151353429503, 36.02624349014384)
       v17/000901-waakzaamheid: (3647.229413364021, 37.06982412600247)
               v17/000963-odin: (3644.5834041964467, 36.69850977273894)
            v17/000907-lincoln: (3640.212359236246, 35.70065870675705)
            v17/000949-warwick: (3635.868500767546, 36.8655737328567)
     v17/000997-golden-phoenix: (3631.7201859899887, 33.768328577501336)
             v17/000817-merlin: (3629.808836540103, 36.096564838158955)
               v17/000976-hope: (3628.5129417317376, 35.788331047139266)

Recent ratings for v17
                v17/000984-ace:  not found
         v17/000985-athenienne:  not found
            v17/000986-theseus:  not found
             v17/000987-superb:  not found
              v17/000988-tiger:  not found
         v17/000989-prometheus:  3619.60 (35.891)
             v17/000990-pallas:  3625.35 (36.017)
         v17/000991-clove-tree:  3498.33 (35.307)
          v17/000992-audacious:  not found
          v17/000993-temeraire:  not found
           v17/000994-hercules:  not found
           v17/000995-intrepid:  3578.89 (30.080)
             v17/000996-merlin:  not found
     v17/000997-golden-phoenix:  3631.72 (33.768)
            v17/000998-revenge:  not found
    v17/000999-royal-sovereign:  not found
              v17/001000-crown:  3611.67 (36.023)
             v17/001001-lively:  not found
            v17/001002-zealous:  not found
          v17/001003-leviathan:  3572.38 (30.565)

real	1m34.027s
user	1m36.237s
sys	0m22.787s
```

### CODE REVIEW

File: ratings/cbt_ratings.py [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+"""
+Build local database of eval games using Google Cloud Bigtable as the source of
+truth.
+
+Inspiration from Seth's experience with CloudyGo db.
+Replacing the much slower and clunkier ratings.py
+
+Usage:
+
+$ export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=/usr/share/grpc/roots.pem
```
Comment (tommadams) [inline ratings/cbt_ratings.py]: Can you link to a doc explaining what `roots.pem` is please?

File: ratings/cbt_ratings.py [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+
+import sys
+sys.path.insert(0, '.')
+
+import re
+import math
+import sqlite3
+from collections import defaultdict, Counter
+
+import choix
```
Comment (tommadams) [inline ratings/cbt_ratings.py]: add choix to requirements.txt

File: ratings/cbt_ratings.py [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+
+        # Both v12 and v15 have the same name for both these models!
+        # 2018-09-30/1538304073-minigo-cc-evaluator-357-332-wb-9bjnq-0-000332-zealous-000357-arachne
+        return None
+
+    run_1, num_1, run_2, num_2 = simple.groups()
+
+    # We have to unravel a mystery here.
+    # filename tells up which number goes with which run,
+    # filename also tells us model_name (number + ame),
```
Comment (tommadams) [inline ratings/cbt_ratings.py]: I think "ame" is a typo

File: ratings/cbt_ratings.py (hunk 1 above)
Comment (sethtroisi) [reply ratings/cbt_ratings.py]: It's only needed for Google's on corp workstations I've removed it.

File: ratings/cbt_ratings.py (hunk 2 above)
Comment (sethtroisi) [reply ratings/cbt_ratings.py]: added to requirements-analysis.txt

**Pull request merged**